CHANGELOG.rst

```diff
@@ -5,13 +5,27 @@
 
 Renku ``0.21.0`` brings the ability to handle expiring Gitlab tokens in user sessions and various bug fixes. 
 
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+* 🎨 **UI**: Improve layout on different pages: project creation, starting sessions, file browser.
+
+**🐞 Bug Fixes**
+
+* 📃 **UI**: Update broken links local projects, to documentation, and add more links to useful resources.
```

Comment:
`Update broken links to local projects and documentation and add more links to useful resources.`

?